### PR TITLE
Added Minecraft compose file

### PR DESCRIPTION
**Pull request by gtardif**

### CODE REVIEW

File: minecraft/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+minecraft_1  | [15:06:39] [Server thread/INFO]: Thread RCON Listener started
+minecraft_1  | [15:06:39] [RCON Listener #1/INFO]: RCON running on 0.0.0.0:25575
+```
+
+Once it is initialized, run your Minecraft application, hit "Play", then "Multiplayer" and "Add server". Specify your new server IP : localhost:25565. You can then start playing.
+
+## Stop and remove the containers
+```
+$ docker-compose down
+```
```
Comment (glours) [inline minecraft/README.md:34]: Hello @gtardif 👋 
Can you follow the sample README template please?
https://github.com/docker/awesome-compose/blob/master/gitea-postgres/README.md

**Comment by glours** [comment]
And add your sample in the [main README ](https://github.com/docker/awesome-compose/blob/master/README.md)

File: minecraft/README.md (hunk 1 above)
Comment (gtardif) [reply minecraft/README.md:34]: Yep sure, missed that going  too quickly !

File: minecraft/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+...
+minecraft_1  | [15:06:39] [Worker-Main-6/INFO]: Preparing spawn area: 94%
+minecraft_1  | [15:06:39] [Worker-Main-7/INFO]: Preparing spawn area: 94%
+minecraft_1  | [15:06:39] [Server thread/INFO]: Time elapsed: 25620 ms
+minecraft_1  | [15:06:39] [Server thread/INFO]: Done (35.526s)! For help, type "help"
+minecraft_1  | [15:06:39] [Server thread/INFO]: Starting remote control listener
+minecraft_1  | [15:06:39] [Server thread/INFO]: Thread RCON Listener started
+minecraft_1  | [15:06:39] [RCON Listener #1/INFO]: RCON running on 0.0.0.0:25575
+```
+
```
Comment (glours) [inline minecraft/README.md:54]: Add the following section.
I can't manage to correctly add code section inside the github suggestion section ¯\\_(ツ)_/¯ 
So the `$ docker ps ...` should be in code section 😅 
```suggestion
Check containers are running and the port mapping:
$ docker ps
CONTAINER ID        IMAGE                   COMMAND             CREATED             STATUS                   PORTS                                 NAMES
7f696c2fb101        itzg/minecraft-server   "/start"            5 minutes ago       Up 5 minutes (healthy)   0.0.0.0:25565->25565/tcp, 25575/tcp   minecraft_minecraft_1
```

File: minecraft/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Once it is initialized, run your Minecraft application, hit "Play", then "Multiplayer" and "Add server". Specify your new server IP : localhost:25565. You can then start playing.
+
```
Comment (glours) [inline minecraft/README.md:56]: Can you add screenshot of server configuration in the Minecraft 🤗?

File: minecraft/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+## Stop and remove the containers
+```
+$ docker-compose down
+```
```
Comment (glours) [inline minecraft/README.md:60]: Can you replace with this ?

> Stop and remove the containers
> 
> ```
> $ docker-compose down
> ```
> 
> To delete all data, remove all named volumes by passing the `-v` arguments:
> ```
> $ docker-compose down -v
> ```

**Comment by 39r762193i** [comment]
> Just few changes to merge the PR

File: minecraft/README.md (hunk 2 above)
Comment (gtardif) [reply minecraft/README.md:62]: done

File: minecraft/README.md (hunk 3 above)
Comment (gtardif) [reply minecraft/README.md:73]: done

File: minecraft/README.md (hunk 4 above)
Comment (gtardif) [reply minecraft/README.md]: done

File: minecraft/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+ Specify your new server IP : localhost:25565
+ ![server configuration](screenshots/add-server-config.png)
+
+ You can then start playing
+ ![ready to play](screenshots/ready-to-play.png)
+
+Stop and remove the containers
+
+$ docker-compose down
+To delete all data, remove all named volumes by passing the -v arguments:
```
Comment (glours) [inline minecraft/README.md:75]:
```suggestion
$ docker-compose down  
To delete all data, remove all named volumes by passing the -v arguments:
```

**Pull request merged**